src/components/form-slice/FormPartUpstream/index.tsx

```diff
@@ -174,37 +182,40 @@ export const FormSectionTimeout = () => {
 export const FormSectionKeepAlive = () => {
   const { t } = useTranslation();
   const { control } = useFormContext<FormPartUpstreamType>();
+  const np = useNamePrefix();
   return (
     <FormSection legend={t('form.upstream.keepalivePool.title')}>
       <FormItemNumberInput
         control={control}
-        name="keepalive_pool.size"
+        name={np('keepalive_pool.size')}
         label={t('form.upstream.keepalivePool.size')}
       />
       <FormItemNumberInput
         control={control}
-        name="keepalive_pool.idle_timeout"
+        name={np('keepalive_pool.idle_timeout')}
         label={t('form.upstream.keepalivePool.idleTimeout')}
         suffix="s"
       />
       <FormItemNumberInput
         control={control}
-        name="keepalive_pool.requests"
+        name={np('keepalive_pool.requests')}
         label={t('form.upstream.keepalivePool.requests')}
         allowDecimal={false}
       />
     </FormSection>
   );
 };
+
 export const FormPartUpstream = () => {
   const { t } = useTranslation();
+  const np = useNamePrefix();
   return (
     <>
       <FormPartBasic />
       <FormSection legend={t('form.upstream.findUpstreamFrom')}>
         <FormSection legend={t('form.upstream.nodes.title')}>
           <FormItemNodes
-            name="nodes"
+            name={np('nodes')}
```

Comment:
I can only delete this by clicking delete twice.

<img width="748" alt="image" src="https://github.com/user-attachments/assets/a8a43dde-9f67-4cef-a9da-7351cb07cc64" />

Reply:
done